dask/array/image.py

```diff
@@ -0,0 +1,55 @@
+from .core import Array
+from glob import glob
+import os
+try:
+    from skimage.io import imread as sk_imread
+except ImportError:
+    pass
+
+
+def tokenize(*args):
+    from hashlib import md5
+    return md5(str(args).encode()).hexdigest()
+
+def add_leading_dimension(x):
+    return x[None, ...]
+
+
+def imread(filename):
+    """ Read a stack of images into a dask array
```

Comment:
I had the good fortune to run into @stefanv just a moment ago.  He noted that `skimage` is more likely to be installed than `imageio` at the moment but that this would change once `skimage` depends on `imageio`.  It might be good for `dask.array` to follow `skimage` in switching, if it switches.